client/components/status/status.js

```diff
@@ -276,6 +279,15 @@ export default class Status extends React.Component {
     this.setState({screenWidth: window.innerWidth});
   }
 
+  updateSpinner = () => {
+    const {activeSessions, passedSessions} = this.state;
+    if (activeSessions.length === 0 && passedSessions.length === 0) {
+      this.setState({loadSpinner: false});
+    } else {
+      this.setState({loadSpinner: true});
+    }
```

Comment:
what about:

```javascript
this.setState({
  loadSpinner: activeSessions.length || passedSessions.length
});
```

Or anything along these lines? The if/else is not really necessary since we're setting a boolean, the result of the set can be the if/else condition.